packages/server/lib/controllers/proxy.coffee

```diff
@@ -312,14 +300,14 @@ module.exports = {
       if remoteState.strategy is "file" and req.proxiedUrl.startsWith(remoteState.origin)
         opts.url = req.proxiedUrl.replace(remoteState.origin, remoteState.fileServer)
       else
-        opts.url = remoteUrl
-
-      ## if we have auth headers and this request matches our origin policy
-      if (a = remoteState.auth) and resMatchesOriginPolicy()
-        ## and no existing Authentication headers
-        if not req.headers["authorization"]
-          base64 = Buffer.from(a.username + ":" + a.password).toString("base64")
-          req.headers["authorization"] = "Basic #{base64}"
+        opts.url = req.proxiedUrl
+
+      ## if we have auth headers and this request matches our origin protection space
```

Comment:
When we are fixing edge cases like this, always try to remember to leave a comment point to the github issue that it represents. You usually do, just pointing it out.